code/game/gamemodes/changeling/changeling_powers.dm

```diff
@@ -143,6 +144,10 @@ var/global/list/possible_changeling_IDs = list("Alpha","Beta","Gamma","Delta","E
 	var/datum/changeling/changeling = changeling_power(0,0,100)
 	if(!changeling)	return
 
+	if (changeling.justate + 60 SECONDS > world.time)

```

Comment:
timer might be a better way

Reply:
A timer is more complex and more overhead for 0 benefit. It's just a cooldown, it doesn't need to do something when it clears.